casper/src/test/scala/coop/rchain/casper/util/comm/TransportLayerTestImpl.scala

```diff
@@ -2,41 +2,35 @@ package coop.rchain.casper.util.comm
 
 import cats.Monad
 import cats.effect._
-import cats.mtl.MonadState
+import cats.effect.concurrent.Ref
 import cats.syntax.all._
 import coop.rchain.casper.util.comm.TestNetwork.TestNetwork
 import coop.rchain.comm.CommError.CommErr
 import coop.rchain.comm.protocol.routing._
 import coop.rchain.comm.rp.ProtocolHelper.protocol
 import coop.rchain.comm.transport._
 import coop.rchain.comm.{CommError, PeerNode}
-import coop.rchain.shared.AtomicMonadState
-import monix.execution.atomic.AtomicAny
 
 import scala.collection.immutable.Queue
 import scala.concurrent.duration.FiniteDuration
 
 object TestNetwork {
   type NodeMessageQueues = Map[PeerNode, Queue[Protocol]]
-  type TestNetwork[F[_]] = MonadState[F, NodeMessageQueues]
+  type TestNetwork[F[_]] = Ref[F, NodeMessageQueues]
 
   def apply[F[_]](implicit ev: TestNetwork[F]): TestNetwork[F] = ev
 
   def addPeer[F[_]: TestNetwork](peer: PeerNode): F[Unit] =
-    TestNetwork[F].modify(t => t + (peer -> Queue.empty))
+    TestNetwork[F].update(t => t + (peer -> Queue.empty))
 
   def peerQueue[F[_]: TestNetwork: Monad](peer: PeerNode): F[Queue[Protocol]] =
-    TestNetwork[F].inspect(_(peer))
+    TestNetwork[F].get.map(_(peer))
```

Comment:
Monad seems unused here.

Reply:
> Monad seems unused here.

We need `Functor` to make `map` operation. It requires one more import. So better to leave `Monad` as is.